MANIFEST.in

```diff
@@ -1,3 +1,3 @@
 include LICENSE
 include requirements/base.txt
-recursive-include tests *.py
+recursive-include tests *.py *.pickle
```

Comment:
No time to check now, what does this do?

Reply:
Adding the pickle files to the sdist (in an attempt to ensure the tests will continue to work when the sdist is installed)